client/src/components/landing/Landing.js

```diff
@@ -1,32 +1,64 @@
 import React, { useState } from 'react';
-import { Card } from 'antd';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
 
-import LoginForm from './LoginForm';
-import RegisterForm from './RegisterForm';
-import './Landing.css';
+import { authActions } from '../../sagas/actions';
 
-const Landing = props => {
-  const tabList = [
-    { key: 'login', tab: 'Login' },
-    { key: 'register', tab: 'Register' },
-  ];
+const Landing = ({ register }) => {
+  const [formData, setFormData] = useState({ id: '', name: '' });
 
-  const [tabKey, setTabKey] = useState('login');
   return (
-    <div className='landing'>
-      <div className='landing-content'>
-        <h1>Welcome to Cribbage!</h1>
-        <Card
-          title='Login or register to play cribbage against your friends online'
-          tabList={tabList}
-          activeTabKey={tabKey}
-          onTabChange={k => setTabKey(k)}
-        >
-          {tabKey === 'login' ? <LoginForm /> : <RegisterForm />}
-        </Card>
-      </div>
+    <div className='max-w-sm m-auto mt-4'>
+      <h1 className='text-4xl'>Welcome to Cribbage!</h1>
+      <p>Play cribbage against your friends online. Get started now!</p>
+      <form
+        onSubmit={e => {
+          e.preventDefault();
+          register(formData);
+        }}
+      >
+        <input
+          onChange={e => setFormData({ ...formData, id: e.target.value })}
+          value={formData.id}
+          placeholder='Username'
+          required
+          className='mt-2 pl-2 h-8 shadow-sm rounded-lg block w-full focus:outline-none focus:shadow-md'
```

Comment:
:oof: maybe I've just been out of the client world for too long. Is this _really_ the best way to define a classname? is there no other way?

I'll keep reading on tailwind to find out..

Reply:
Tailwind provides a number of utility classes which are combined to create the overall effect desired. What will help with these inputs is to extract the common classes into a component as I've done for the `btn` and `btn-primary` classes.

Reply:
https://tailwindcss.com/docs/extracting-components